lib/index.js

```diff
@@ -20,8 +21,12 @@ function positiveNumber (number) {
 Asayer.prototype.initialize = function () {
   var projectId = positiveNumber(this.options.projectId)
   if (!projectId) {
-    console.warn('Asayer wrong projectId option, not loading.')
-    return
+    // Fall back to siteId.
+    projectId = positiveNumber(this.options.siteId)
```

Comment:
Alternatively, instead of adding a new Segment developer center app setting for `projectId`, it may be cleaner to just update the label for the existing setting `siteId` and this code only use `siteId`.